draft-irtf-cfrg-frost.md

```diff
@@ -1250,11 +1250,13 @@ prime-order group.
 One possible key generation mechanism is to depend on a trusted dealer, wherein the
 dealer generates a group secret `s` uniformly at random and uses Shamir and Verifiable
 Secret Sharing as described in {{dep-shamir}} and {{dep-vss}} to create secret
-shares of `s` to be sent to all other participants. We highlight at a high level how this
-operation can be performed.
+shares `s_1, ..., s_MAX_SIGNERS` of `s` to be sent to all `MAX_SIGNERS` participants.
+This operation is specified in the `trusted_dealer_keygen` algorithm. The mathematical relation
+between the secret key `s` and the `n` secret shares is formalized in the `secret_share_combine(shares)`
+algorithm, defined in {{dep-shamir}}. Note that in the setting of thrshold signatures, this algorithm is not needed, as signers
+can generate signature shares locally without having to disclose their secret share.
```

Comment:
I get what this is trying to say, but it seems strange because this document exists in the threshold setting. Maybe this should say in settings without a trusted dealer instead?

Reply:
I can just remove the "Note that...." entirely? I mainly wanted to say that we define `secret_share_combine` as a tool for understanding the mathematical structure, and it is not intended to actually be used anywhere in the draft. If you have a better way to say that, definitely open to suggestions.

Reply:
Yeah, I might just remove that sentence.